gfx.cpp

```diff
@@ -4786,7 +4787,7 @@ class GfxInternal
                 uint32_t const window_width  = GFX_MAX(window_rect.right,  (LONG)8);
                 uint32_t const window_height = GFX_MAX(window_rect.bottom, (LONG)8);
                 fence_index_ = swap_chain_->GetCurrentBackBufferIndex();
-                if(window_width != window_width_ || window_height != window_height_)
+                if(!IsIconic(window_) && (window_width != window_width_ || window_height != window_height_))
```

Comment:
This ignores resize wen minimised. This prevents user apps from triggering a resize to 8x8 (and issues around that) only to hav to resize back again when returning from minimised state